### DISCUSSION TITLE
Support for Grammy-inline-menu

### DISCUSSION THREAD

**Discussion by felinto-dev** [bukhalo/nestjs-telegraf#564] [Ideas]
I would like to ask you to add support for grammy-inline-menu. For small projects, nestjs-telegraf works very well, but as new menus are created it is almost impossible to manage.

**Comment by felinto-dev** [comment]
I would just like to let you know that I am developing a library for nestjs-telegraf that will help a lot in creating complex menus using NestJS decorators. Once it's done I'll post it here!

**Comment by filipmacek** [comment]
Yes, was also looking how to implement this or is it possbile

**Comment by filipmacek** [comment]
Grammy-inline-menu started using grammy instead of telegraf as their framework.
But this project looks good and it uses telegraf.
https://github.com/DrSmile444/telegraf-menu
